platform/web/serve.py

```diff
@@ -38,12 +38,21 @@ def shell_open(url):
 def serve(root, port, run_browser):
     os.chdir(root)
 
+    address = ("", port)
+    httpd = DualStackServer(address, CORSRequestHandler)
+
     if run_browser:
         # Open the served page in the user's default browser.
         print("Opening the served URL in the default browser (use `--no-browser` or `-n` to disable this).")
         shell_open(f"http://127.0.0.1:{port}")
```

Comment:
Print the URL that is being served, so that it can be manually copied in a web browser if needed:

```suggestion
    url = f"http://127.0.0.1:{port}"
    if run_browser:
        # Open the served page in the user's default browser.
        
        print(f"Opening the served URL in the default browser (use `--no-browser` or `-n` to disable this): {url}")
        shell_open(url)
    else:
        print(f"Serving at: {url}")
```